tasks/Minimum-Effort Tacit Coordination Game

```diff
@@ -0,0 +1,43 @@
+# Summary
+> Players are provided different equilibrium points that are pareto ranked and need to make a payoff decision that reaches a efficient equibilibrium point with uncertainty testing payoff dominance.  
+
+# References
+> Original Paper: https://www.jstor.org/stable/2006745?casa_token=8Kk87aHnoVIAAAAA%3AuNhlcFLseFbinYshXbx56f0Jt6jUyJW_62oZgXV14PXEcbwzwCV4MebPnUimUi9Qz4wgn7MOcAw6T6ZkjYKNzc0Q3MMcQBRj3LJKnHShXISvBOAGeLls&seq=1
+
+> Van Huyck, J., Battalio, R., & Beil, R. (1990). Tacit Coordination Games, Strategic Uncertainty, and Coordination Failure. The American Economic Review, 80(1), 234-248. Retrieved March 10, 2021, from http://www.jstor.org/stable/2006745
+
+> Crawford, Vincent P., "An 'Evolutionary' Interpretation of Van Huyck, Battalio, and Beil's Experimental Results on Coordination," unpublished manuscript,   January 1989
+ 
```

Comment:
Should the Frontiers paper (https://www.frontiersin.org/articles/10.3389/fpsyg.2019.00112/full#B91) also be analyzed?

Reply:
I'll add this in as a reference paper! Thank you!